Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Cultural GFX Improvements

## Changelog:
### Graphical Cultures

Comprehensive CoA and Building cultural graphics overhaul for almost every culture group (Orcs notable exception). 
  - Clothing should be addressed in a separate PR.
  - New CoA gfx assembled using vanilla emblems; further additions of custom wc emblem .dds is optimal.
  
<details>
<summary> Click to Show: Added custom CoA gfx:</summary>

  - Ancient (Pagan symbols/Tree/Animal motifs)
  - Dryad (Lunar and animal motifs)
  - Night Elf (Lunar and animal motifs)
  - Highborne and Nightborne (Lunar motifs)
  - Harpy (African pagan and avian motifs)
  - Arakkoa (African pagan and avian motifs)
  - Furbolg (Animal motifs)
  - Gnoll (Pagan symbols and feudal Animal heraldry)
  - Kobold (Pagan symbols and feudal Animal heraldry)
  - Quilboar (Pagan african and animal motifs)
  - Tauren (Pagan symbols and animal motifs)
  - Trogg (Pagan symbols and feudal Animal heraldry)
  - Tuskar (Scandanavian (Not Viking) lists, Pagan symbols and feudal Animal heraldry)
 Note[^3]

</details>

<details>
<summary> Click to Show: New vanilla CoA additions: </summary> 

  - Centaur (Steppe)
  - Draenei (Zoroastrianism)
  - Dragon (Byzantine and Welsh)
  - Murloc (African Pagan)
  - Quilboar (African Pagan)
  - Titanforged (Byzantine)
  - Titan (Byzantine)
Note[^4]

</details>
  
<details>
<summary> Click to Show: Added building gfx: </summary>
 
- Indian[^1]
    - Ancient
    - Dryad
    - Harpy 
    - Draenei
    - Satyr
    - Furbolg
    - Murloc
    - Tauren
    - Highmountain
- Steppe[^2]
    - Centaur
    - Furbolg
    - Tauren
    - Grummle
- African[^2]
    - Naga
    - Gnoll
    - Kobold 
    - Quilboar
    - Highmountain
    - Trogg
    - Vulpera
- MENA
    - Centaur
    - Aqir 
    - Arakkoa
    - Gnoll
    - Goblin
    - Quilboar
    - Shathyar
    - Tolvir
    - Tortollan
    - Trogg
    - Vulpera
- Mediterranean
    - Dragon
    - Elves
    - Naga
    - Tolvir
    - Titanforged Group (except Tolvir)
- Norse
    - Taunka
    - Tuskar
Note:[^5]

</details>

<details>
<summary> Click to Show: Updated unit gfx </summary>

  - Centaur -> mongol 
  - Tuskar -> northern

</details>

<details>
<summary>Click to Show: Updated clothing gfx:</summary>

  - Tuskar -> northern 

</details>

### Misc
- Moved Kirin Tor emblem from Arcane Group to Kirin Tor faith
- Males of the Centaur Group and Dryad Group are now topless.


## Tests:
- [x] Elunic_coa_gfx not working ; refer to [9c6039a](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/868/commits/9c6039a9fd80ed4dbcbe1896a5f5b5d4ad5bc7d3)


# How to test:

Check for correct CoA assignments for each affected culture in the changelog. 

[^1]: Indian Tribal buildings are stilted, wooden constructions that are relatively exposed. ,making them suited to airborne, aquatic and more primitive tribal cultures.
[^2]: African and Steppe gfx only apply in the Tribal Era, hence why cultures with them also have another gfx for when/if they feudalise.
[^3]: Some of these cultures share custom gfx.
[^4]: Some of these use repurposed emblem lists.
[^5]: Requires Northern Lords DLC
